src/AspNetCoreModuleV2/RequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -807,7 +813,9 @@ IN_PROCESS_APPLICATION::ExecuteApplication(
 
     DBG_ASSERT(m_status == APPLICATION_STATUS::STARTING);
 
-    hModule = LoadLibraryW(m_pConfig->QueryHostFxrFullPath());
+    // hostfxr should already be loaded by the shim. If not, then we will need
+    // to load it ourselves by finding hostfxr again.
+    hModule = LoadLibraryW(L"hostfxr.dll");
```

Comment:
const?

Reply:
Which portion do you want to be const? The string itself is const already, I can make HMODULE const too.